content/spin/serverless-ai-tutorial.md

```diff
@@ -232,6 +232,14 @@ Now let's use the Spin SDK to access the model from our app:
 
 {{ startTab "Rust"}}
 
+> Note: The following Rust source code, for this sentiment analysis example, uses serde. Please add the following dependencies to your Rust application's `Cargo.toml` file before you build/run the application.
```

Comment:
The line that led into this tab block (line 229) and the section title (line 227) seemed to be leading directly into the source code.  With this note here the colon at the end looks amiss (though it works with the TypeScript tab).  I am not sure this will bother anybody else though!